app/manifest/api_settings/endpoint_settings.go

```diff
@@ -29,29 +30,38 @@ func (setting EndpointSettings) Valid() validation.ValidateResult {
 		result.AddError(msg)
 	}
 
-	if setting.Route[0] != '/' {
-		var msg = fmt.Sprintf("api.endpoints[%s].route should start with '/' ex: /api/hello", setting.ActionID)
+	if len(setting.ActionID) == 0 && len(setting.FlowActionID) == 0 {
+		var msg = fmt.Sprintf("Should be informed an api.endpoints[%s].actionId or api.endpoints[%s].flowActionId", setting.Route, setting.ActionID)
 		result.AddError(msg)
 	}
 
-	if len(setting.Method) == 0 {
-		var msg = fmt.Sprintf("api.endpoints[%s].method is required", setting.ActionID)
-		result.AddError(msg)
-	} else {
-		if (setting.Method == types.HttpMethodGet ||
-			setting.Method == types.HttpMethodPost ||
-			setting.Method == types.HttpMethodPut ||
-			setting.Method == types.HttpMethodPatch ||
-			setting.Method == types.HttpMethodDelete) == false {
+	if !setting.IsProxy {
 
-			var msg = fmt.Sprintf("api.endpoints[%s].method should contain valid value (get, post, put, patch or delete)", setting.ActionID)
+		if setting.Route[0] != '/' {
+			var msg = fmt.Sprintf("api.endpoints[%s].route should start with '/' ex: /api/hello", setting.ActionID)
 			result.AddError(msg)
 		}
-	}
 
-	if len(setting.ActionID) == 0 && len(setting.FlowActionID) == 0 {
-		var msg = fmt.Sprintf("Should be informed an api.endpoints[%s].actionId or api.endpoints[%s].flowActionId", setting.Route, setting.ActionID)
-		result.AddError(msg)
+		if len(setting.Method) == 0 {
```

Comment:
Is working with "else" a semantic decision? If not, perhaps using if only could improve code readability over time.